code/_onclick/hud/screentip.dm

```diff
@@ -2,21 +2,13 @@
 	icon = null
 	icon_state = null
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
-	screen_loc = "TOP,LEFT"
-	maptext_height = 480
-	maptext_width = 480
+	screen_loc = "TOP,CENTER-1"
+	maptext_height = 128
+	maptext_width = 128
```

Comment:
is this just to make it look right? or

Reply:
ayup, it starts in a weird spot otherwise.